.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -16,18 +22,18 @@ ASO V1: `kubectl get deployment -n azureoperator-system azureoperator-controller
 ASO V2: `kubectl get deployment -n azureserviceoperator-system azureserviceoperator-controller-manager -o wide` and share the image being used by the manager container.   
 -->
 
-**Describe the bug**
-A clear and concise description of what the bug is.
+## To Reproduce
+
+_Describe the steps to recreate the issue._
+
+## Expected behavior
```

Comment:
We had this before too but I'm always a bit torn by this section because it feels duplicate from "Describe the bug" above. How do you describe the bug without discussing what the expected behavior was?

Happy to keep this though but maybe put it above reproduction steps?